sentry-core/src/main/java/io/sentry/core/protocol/Browser.java

```diff
@@ -28,9 +31,29 @@ public void setVersion(String version) {
     this.version = version;
   }
 
+  @TestOnly
+  Map<String, Object> getUnknown() {
+    return unknown;
+  }
+
   @ApiStatus.Internal
   @Override
   public void acceptUnknownProperties(Map<String, Object> unknown) {
     this.unknown = unknown;
   }
+
+  /**
+   * Clones a Browser aka deep copy
+   *
+   * @return the cloned Browser
+   * @throws CloneNotSupportedException if object is not cloneable
+   */
+  @Override
+  public @NotNull Browser clone() throws CloneNotSupportedException {
+    final Browser clone = (Browser) super.clone();
+
+    clone.unknown = CollectionUtils.shallowCopy(unknown);
```

Comment:
I just realized that if unknown isn't thread safe this could throw. Maybe we should guard all of these clones?

Reply:
Take a look please at https://github.com/getsentry/sentry-android/pull/504#discussion_r463582852